dgraph/cmd/zero/raft.go

```diff
@@ -997,21 +998,23 @@ func (n *node) Run() {
 				}
 				n.Applied.Done(entry.Index)
 			}
-			span.Annotatef(nil, "Applied %d CommittedEntries", len(rd.CommittedEntries))
+			span.AddEvent(fmt.Sprintf("Applied %d CommittedEntries", len(rd.CommittedEntries)))
 
 			if !leader {
 				// Followers should send messages later.
 				for i := range rd.Messages {
 					n.Send(&rd.Messages[i])
 				}
 			}
-			span.Annotate(nil, "Sent messages")
+			span.AddEvent("Sent messages")
 			timer.Record("proposals")
 
 			n.Raft().Advance()
-			span.Annotate(nil, "Advanced Raft")
+			span.AddEvent("Advanced Raft")
 			timer.Record("advance")
 
+			span.SetAttributes(attribute.String("Zero.RunLoop", "Running"))
```

Comment:
why are we setting this attribute as Done right after we are setting it as Running?